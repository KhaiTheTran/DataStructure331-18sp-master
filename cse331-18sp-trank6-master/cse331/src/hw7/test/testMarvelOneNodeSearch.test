# Test Dijkstra's algorithm with an one node file

#Load graph
LoadGraph nodeone nodeone.tsv

# check graph name
ListNodes nodeone
ListChildren nodeone KILLRAVEN/JONATHAN_R

# Test Dijkstra's algorithm
FindPath nodeone KILLRAVEN/JONATHAN_R KILLRAVEN/JONATHAN_R
FindPath nodeone KILLRAVEN/JONATHAN_R node_a
FindPath nodeone node_b KILLRAVEN/JONATHAN_R