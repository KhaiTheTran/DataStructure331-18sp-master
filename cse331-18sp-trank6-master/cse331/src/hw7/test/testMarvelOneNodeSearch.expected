# Test Dijkstra's algorithm with an one node file

#Load graph
loaded graph nodeone

# check graph name
nodeone contains: KILLRAVEN/JONATHAN R
the children of KILLRAVEN/JONATHAN R in nodeone are:

# Test Dijkstra's algorithm
path from KILLRAVEN/JONATHAN R to KILLRAVEN/JONATHAN R:
total cost: 0.000
unknown character node a
unknown character node b
